src/styles/blocks/card.scss

```
@@ -0,0 +1,55 @@
+.product-container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: $card-width;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 11px;
+  font-weight: 400;
+  &__img {
+    display: block;
+    margin: 0 auto 45px;
+    width: 160px;
+    height: 134px;
+  }
+  &__card-header {
+    font-weight: 500;
+    line-height: 15px;
+    font-size: 12px;
+    margin: 0 auto 6px;
+  }
+  &__code {
+    padding-bottom: 15px;
+  }
+  .rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 20px;
+    &__stars {
+      display: flex;
+      .stars__star {
+        width: 15px;
+        height: 15px;
+        margin-right: 4px;
+        background-image: url("../images/star-active.svg");
+        background-repeat: no-repeat;
+        background-position: center;
+        &:last-child {
+          background-image: url("../images/star.svg");
+          margin-right: 0;
+        }
+      }
+    }
+  }
+}
+
+.product-container--wrap {
+  margin: 0 $card-gap 50px 0;
```

```suggestion
 margin-top: 50px;
```
You shouldn't use margin for left and right sides because you have calculated value - width for .main__container